### DISCUSSION TITLE
The number of matching results can be displayed during the search

### DISCUSSION THREAD

**Discussion by Ekkone** [sumatrapdfreader/sumatrapdf#3347] [Ideas]
It is important to be able to display the total number of matches and the current number

**Comment by GitHubRulesOK** [comment]
press A and it will be less than press E in most English words, 
There is little point in counting such characters since PDFs are based on digits and what you see as searchable letters are not the letter on screen, its how many digits are encoded as lower case e, thus its possible to search one ahead or one back based on a digital match.

Here is what a PDF holds as text
<5A61706F7769616461> Tj
/F3 8.89 Tf
<23> Tj
/F1 648.20 Tf
1.1317 0 0 1 98.08 295.61 Tm
<20> Tj
/F1 8.41 Tf
1.3160 0 0 1 107.32 295.61 Tm
<7369> Tj

So in this case <20> is a normal Polish space and the <23> (# in an English file) should be perhaps Polish `ł`  after an incomplete `Zapowiada`  and this is a better than average file as its almost American but in another language it could be a Chinese ideogram as a word. 

However those digits can be wrong so e = A if the mapping is corrupt again counting those is pointless. Its even worse in eastern cases, since PDF was designed for Anglo American fonts. The best way to count words is export those numbers into Plain text and paste in a word processor, where it counts true whitespaces between words (PDF has no concept of words just correctly encoded letters)
